uwrt_mars_rover_hw/include/uwrt_mars_rover_hw/uwrt_mars_rover_hw_control_loop.h

```diff
@@ -36,7 +36,7 @@ class MarsRoverHWControlLoop {
   ros::Time last_control_loop_time_;
   double controller_watchdog_timeout_{0};
   double control_freq_{0};
-  static constexpr double DEFAULT_CONTROL_FREQUENCY_{50.0};
-  static constexpr double DEFAULT_CONTROLLER_WATCHDOG_TIMEOUT_{5.0};
+  static constexpr double DEFAULT_CONTROL_FREQUENCY{50.0};
+  static constexpr double DEFAULT_CONTROLLER_WATCHDOG_TIMEOUT{5.0};
```

Comment:
no underscores for constexpr?

Reply:
I can't get it to work for only member constexpr :sob: You can try to get it to work if you want lol, but i give up.